firebaseFunctions/src/helpers.ts

```diff
@@ -159,6 +163,99 @@ export const updateDealUpdatesCollection = (dealId: string): void => {
   });
 };
 
+/**
+ * DeepDAO api returns mixed avatar URL, which in some cases contains
+ * the full URL and in others return only the file name.
+ * This function unify urls to always contain the full URL path.
+ * @param url string
+ * @returns string
+ */
+const unifyAvatarUrl = (url: string): string => {
+  const pathParts = url?.split("/");
+  return (!url || url.includes("https://"))
+    ? url || ""
+    : DEEP_DAO_ASSETS_URL + pathParts[pathParts.length - 1];
+};
+
+/**
+ * Imports list of DAO's (Organizations) from DeepDAO API, and stores a map
+ * of organizations data inside a Google FireStore collection - `deep-dao`.
+ * The API-Url and API-Key are specified in `firebaseFunctions/.env` file
```

Comment:
> How does adding the url and keys work for firebaseFunctions/.env exactly?

Could you add some instructions in some readme please?

Reply:
These variables are being embeded to the function during deployment, so no instructions needed (other then having this .env file in place...)

Reply:
And having the .env in place is sth for the readme imo.

Also the instructions you gave in the initial comment should also go into the readme.
I mean, how else would one know how to set up DeepDao feature correctly?

Reply:
Added to a read me file